biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/TemplateTool.scala

```diff
@@ -17,7 +17,7 @@ package nl.lumc.sasc.biopet.core
 import java.io.{File, PrintWriter}
 
 import nl.lumc.sasc.biopet.utils.summary.Summary
-import nl.lumc.sasc.biopet.utils.{ConfigUtils, Question, ToolCommand}
+import nl.lumc.sasc.biopet.utils.{AbstractOptParser, ConfigUtils, Question, ToolCommand}
```

Comment:
That's not possible but all we can do here is:
``` scala
import nl.lumc.sasc.biopet.utils._
```
This will include everything this package. I do think it's better to keep is like it is because this imports are mostly controlled by the IDE anyway. If they become to much it will replace it for the underscore automatically